tests/test_settings.py

```diff
@@ -43,7 +43,7 @@
     custom_mapping=None,
     output_format=OutputFormat.HUMAN_SUMMARY,
     ignore_undeclared=set(),
-    ignore_unused=set(),
+    ignore_unused=default_ignore_unused,
```

Comment:
```suggestion
    ignore_unused=DEFAULT_IGNORE_UNUSED,
```

Reply:
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/b6afa98689e4720130fad434d0ce13e2d2b44a26.